src/connections/DelayedJob.ts

```diff
@@ -14,56 +19,76 @@ type DelayedJobCombinedConnection = {
   delayedJobsHasFailed: boolean;
   bulkUpdateJobsIsLoading: boolean;
   bulkUpdateJobsHasFailed: boolean;
-  updatedJobsResponse?: DelayedJobDto[];
 };
 
-const delayedJobsSelector = (store: ApiDataStore) => store.delayedJobs;
+const delayedJobsSelector = (store: ApiDataStore) =>
+  Object.values(store.delayedJobs ?? {})
+    .map(resource => resource.attributes)
+    .filter(({ isAcknowledged }) => !isAcknowledged);
 
 const combinedSelector = createSelector(
   [delayedJobsSelector, bulkUpdateJobsIsFetching, bulkUpdateJobsFetchFailed],
   (delayedJobs, bulkUpdateJobsIsLoading, bulkUpdateJobsFailure) => ({
-    delayedJobs: Object.values(delayedJobs ?? {}).map(resource => resource.attributes),
+    delayedJobs,
     delayedJobsIsLoading: delayedJobs == null && !bulkUpdateJobsFailure,
     delayedJobsHasFailed: Boolean(bulkUpdateJobsFailure),
     bulkUpdateJobsIsLoading,
-    bulkUpdateJobsHasFailed: bulkUpdateJobsFailure != null,
-    updatedJobsResponse: Object.values(delayedJobs ?? {}).map(resource => resource.attributes as DelayedJobDto)
+    bulkUpdateJobsHasFailed: bulkUpdateJobsFailure != null
   })
 );
 
-const combinedLoad = (connection: DelayedJobCombinedConnection) => {
-  if (!combinedIsLoaded(connection)) {
-    listDelayedJobs();
-  }
-};
-
-const combinedIsLoaded = ({
-  delayedJobs,
-  delayedJobsHasFailed,
-  bulkUpdateJobsIsLoading,
-  bulkUpdateJobsHasFailed
-}: DelayedJobCombinedConnection) =>
-  (delayedJobs != null || delayedJobsHasFailed) && !bulkUpdateJobsIsLoading && !bulkUpdateJobsHasFailed;
-
 const delayedJobsCombinedConnection: Connection<DelayedJobCombinedConnection> = {
-  load: combinedLoad,
-  isLoaded: combinedIsLoaded,
   selector: combinedSelector
 };
 
+export const useJobProgress = () => useSelector<AppStore, JobsDataStore>(({ jobs }) => jobs);
+
 export const useDelayedJobs = () => {
   const connection = useConnection(delayedJobsCombinedConnection);
+  const { totalContent } = useJobProgress();
+  const intervalRef = useRef<NodeJS.Timer | undefined>();
+  const [, { isLoggedIn }] = useLogin();
 
+  const stopPolling = useCallback(() => {
+    if (intervalRef.current != null) {
+      clearInterval(intervalRef.current);
+      intervalRef.current = undefined;
+    }
+  }, []);
+  const startPolling = useCallback(() => {
+    if (intervalRef.current == null) {
+      intervalRef.current = setInterval(() => {
+        listDelayedJobs();
+      }, 1500);
+    }
+  }, []);
```

Comment:
These methods are idempotent - you can call them as many times as you like and if their job is already complete, they simply NOOP. That turns out to be really handy for simplifying the code in the effect below that starts and stops polling based on conditions in the redux store.